salsah/src/public/vendor/ckeditor-4.7.1-Editor/config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+CKEDITOR.editorConfig = function( config ) {
```

Comment:
What is the purpose of this config file and how does it relate to the config provided in `jquery.htmleditor.js`?